test/controllers/patients_controller_test.rb

```diff
@@ -123,6 +123,49 @@ def teardown; end
 
   # ------- Household Updates -----
 
+  test 'head_of_household_options filters out exclude_patient_id from table' do
+    user = create(:public_health_enroller_user)
+    current_patient = create(:patient, creator: user)
+    create(:patient, creator: user)
+    create(:patient, creator: user)
+    create(:patient, creator: user)
+    create(:patient, creator: user)
+
+    sign_in user
+
+    exclude_patient_id = current_patient.id
+    post :head_of_household_options, params: {
+      query: {
+        search: '',
+        entries: 5,
+        workflow: 'global',
+        tab: 'all',
+        scope: 'all',
+        tz_offset: 240,
+        exclude_patient_id: exclude_patient_id,
+        filter: [{
+          dateOption: nil,
+          filterOption: {
+            description: 'Monitorees that are a Head of Household or self-reporter',
+            name: 'hoh',
+            title: 'Daily Reporters (Boolean)',
+            type: 'boolean'
+          },
+          value: true
+        }]
```

Comment:
this `filter` param is used for the advanced filter and it's probably better to not include this when unit testing the logic involving the `exclude_patient_id` param to prevent any potential interference with the logic (not that that's the case right now)